### ISSUE TITLE
mj-image breaks when href attribute contains "=="

### ISSUE THREAD

**Issue by jsbsantos** [mjmlio/mjml#1092]
I was using the API and noticed a strange behavior regarding the mj-image element where sometimes I would get an empty src attribute. I did some tests and eventually went to test it out on your [live test page](https://mjml.io/try-it-live/Sky_gTU5G).

So I used your example there and changed it a bit:
```<mjml>
  <mj-body>
    <mj-container>
      <mj-section>
        <mj-column>
          <mj-image  
                href="http://www.google.com"
                src="http://www.online-image-editor.com//styles/2014/images/example_image.png"
                alt="image is busted"
          ></mj-image>
        </mj-column>
      </mj-section>
    </mj-container>
  </mj-body>
</mjml>
```

This gave the expected result and i got the image with a link. However, when I changed the href attribute to "**http://www.google.com?q=aW1hZ2UgaXMgYnVzdGVkIQ==**" I get an empty src attribute.
```<mjml>
  <mj-body>
    <mj-container>
      <mj-section>
        <mj-column>
          <mj-image  
                href="http://www.google.com/search?q=aW1hZ2UgaXMgYnVzdGVkIQ=="
                src="http://www.online-image-editor.com//styles/2014/images/example_image.png"
                alt="image is busted"
          ></mj-image>
        </mj-column>
      </mj-section>
    </mj-container>
  </mj-body>
</mjml>
```

**Exhibit 1:**
![image](https://user-images.githubusercontent.com/729094/37959015-ec676da6-31a9-11e8-99eb-37aa5d22b284.png)

This happened because the URL I am using has a base64 encoded query string parameter that ends in "==". When I test with the same URL, but remove the "==" instead, it works.

**Exhibit 2:**
![image](https://user-images.githubusercontent.com/729094/37961476-5215e438-31b0-11e8-98ef-a532b256e6ee.png)

I'm not really sure what's happening here but I would appreciate your input.

Thanks.

**Comment by iRyusa** [comment]
Hi @jsbsantos 

Thanks for reporting this ! I see that it's the same behavior on MJML4, can you take a look @kmcb777  ?

**Comment by kmcb777** [comment]
Thanks, turns out this is related to attribute encoding, and is fixed by 5dcbfd0 which will be included in the next release :)

**Comment by kmcb777** [comment]
Hi @jsbsantos we released a new beta for mjml 4.1, which you can install with `npm install mjml@next`. There has been noticeable changes to the parser, could you check if it resolves the problem you had ? (It should)
